css/main.css

```diff
@@ -33,7 +33,7 @@ h3,
 h4,
 h5,
 h6 {
-  font-family: 'Rowdies', sans-serif;
+  font-family: "Rowdies", sans-serif;
```

Comment:
Lots of Prettier fixes mixed in this review.. I'll try to highlight my main changes